src/rent-vs-mortgage.md

```diff
@@ -432,73 +444,70 @@ const deckInstance = new DeckGL({
   ],
   controller: true,
   map: mapboxgl,
-  mapboxApiAccessToken:
-    "pk.eyJ1IjoibWF4Ym8iLCJhIjoiY2lsd2JnZDRyMDFxNHZna3MwZDZmN2R0ZCJ9.32XWATCazaiDzdW6bXvBxw",
-  mapStyle: "mapbox://styles/mapbox/light-v10",
+  mapboxApiAccessToken: 'pk.eyJ1IjoibWF4Ym8iLCJhIjoiY2lsd2JnZDRyMDFxNHZna3MwZDZmN2R0ZCJ9.32XWATCazaiDzdW6bXvBxw',
+  mapStyle: 'mapbox://styles/mapbox/light-v10',
   layers: [
     new GeoJsonLayer({
-      id: "sa2s",
-      data: sa2s,
-      opacity: 0.5,
-      wireframe: true,
+      id: 'sa2s',
+      data: melbourneSa2s,
+      pickable: true,
       getFillColor: (f) => {
-        const sa2 = dataArray.find((d) =>
-          d.sa2_code.toString() === f.properties.SA2_MAIN16
-        );
-
-        if (!sa2) {
-          return undefined;
-        }
+        const sa2 = sa2Map.get(f?.properties?.REGION_CODE)
+        if (!sa2) return
 
-        const hex = aggregateRatiosPlot.scale("color").apply(
-          sa2.repayment_to_rent_ratio,
-        );
+        const hex = deckColours.apply(sa2.repayment_to_rent_ratio)
+        const color = Color(hex)
 
-        const color = Color(hex);
-        return [color.red * 255, color.green * 255, color.blue * 255];
+        return [color.red * 255, color.green * 255, color.blue * 255, 0.6 * 255]
       },
-      getText: (f) => f.properties.SA2_MAIN16,
-      // getLineColor: () => [255, 255, 255],
+      getLineColor: 0,
+      getLineWidth: 20,
     }),
   ],
-  getTooltip: ({object}) => {
-    return object && object.properties.SA2_MAIN16
-  } 
-});
+  getTooltip: (point) => {
+    const properties = point?.object?.properties
+    if (!properties) return
+
+    const sa2 = sa2Map.get(properties.REGION_CODE)
+    return sa2
+      ? `${properties.REGION_NAME}\nMortgage ${percentFormat(sa2.repayment_to_rent_ratio)} of Rent`
+      : properties.REGION_NAME
+  },
+})
 
 // clean up if this code re-runs
 invalidation.then(() => {
-  deckInstance.finalize();
-  container.innerHTML = "";
-});
+  deckInstance.finalize()
+  container.innerHTML = ''
+})
 ```
 
 
-```js
-const sa2Codes = dataArray.map(d => d.sa2_code.toString());
-const melbourneSa2s = {...sa2s, features: sa2s.features.filter(f => sa2Codes.includes(f.properties.SA2_MAIN16))}
+<div class='card'>
 
-view(Plot.plot({
+```js
+Plot.plot({
   aspectRatio: 1,
   color: {
     legend: true,
-    domain: [1, 9],
-    label: "Repayment to rent ratio",
+    domain,
+    label: 'Repayment to rent ratio',
   },
   marks: [
     Plot.geo(melbourneSa2s, {
-      fill: (f) => {
-        const sa2 = dataArray.find((d) =>
-          d.sa2_code.toString() === f.properties.SA2_MAIN16
-        );
-
-        if (!sa2) {
-          return undefined;
-        }
-
-        return sa2.repayment_to_rent_ratio
-      },
+      fill: f => sa2Map.get(f?.properties?.REGION_CODE)?.repayment_to_rent_ratio,
```

Comment:
we might want to add a `stroke: "black"` to this if want the Deck and Plot maps to look the same

Reply:
Yeah, better like this for my tastes.

![image](https://github.com/user-attachments/assets/02a6ae15-e7cf-49cd-ac0c-1a25d2ff2613)